src/scss/layout/_footer.scss

```diff
@@ -1 +1,73 @@
 @use 'abstracts' as *;
+
+.site-footer {
+  @extend %body__text-s;
+  position: relative;
+  display: flex;
+  align-items: center;
+
+  @include breakpoint('medium') {
+    height: rem(183px);
+  }
+
+  &::before {
+    content: '';
+    position: absolute;
+    left: calc((100vw - 100%) / -2);
+    z-index: -1;
+    display: block;
+    width: 100vw;
+    height: 100%;
+    background-color: $color__gray-500;
+  }
+
+  a {
+    color: $color__blue-200;
+
+    &:hover {
+      color: $color__blue-700;
+      text-decoration: none;
+    }
+  }
+
+  &__about {
+    display: grid;
+    grid-column-gap: rem(96px);
+    grid-row-gap: rem(16px);
+    padding: rem(48px) 0;
+  
+    @include breakpoint('medium') {
+      grid-template-columns: rem(310px) rem(400px) rem(310px); 
```

Comment:
Instead of doing this with pixels, I would do it with fractions units. I checked with Tucker, and the note on her mockup is incorrect. You can update this to `3fr 4fr 3fr` to get the correct proportions. Let's also change the breakpoint from medium to large (it gets weird on some larger tablets).